requirements.txt

```diff
@@ -4,3 +4,5 @@ wandb>=0.10.32,<1  # pip will likely update it to 0.12.1, but it is probably ok
 transformers>=4.6.0,<5  # pip will likely update it to 4.10.0, but it is probably ok and good for bugfixes.
 accelerate>=0.4.0,<1  # We may want to use 0.5.0 in the near future
 datasets[streaming]>=1.11.0,<2
+wikipedia2vec==1.0.5
+nltk==3.6.5
```

Comment:
Really a tiny detail, but I think we can make these dependencies optional by adding them to the `setup.py` in `extras_require` :slightly_smiling_face: :
```
setup(
    name="bsmetadata",
    python_requires=">=3.7.11, <3.10",
    version="0.1.0",
    url="https://github.com/bigscience-workshop/metadata.git",
    author="Multiple Authors",
    author_email="xxx",
    description="Codebase for including metadata (e.g., URLs, timestamps, HTML tags) during language model pretraining.",
    packages=find_packages(),
    install_requires=install_requires,
    extras_require={
        "website_description_preprocessing": ["wikipedia2vec==1.0.5", "nltk==3.6.5"],
    },
)
```

Reply:
thank you for pointing this out @SaulLu :) I've made the required changes